### PR TITLE
Update G Suite designs

### PR DESCRIPTION
**Pull request by scruffian**
This updates the G Suite screens with the designs from p6qnuF-4HE-p2

screen shot 2017-05-25 at 17 26 09
screen shot 2017-05-25 at 17 25 37
screen shot 2017-05-25 at 17 25 14
screen shot 2017-05-25 at 17 24 55
  
#### Testing instructions
  
1. Run `git checkout update/g-suite` and start your server
2. Open the [`Email page` page](http://calypso.localhost:3000/domains/manage/[domain]/email/[domain])
3. Check that the designs look as above
2. Go through the domain purchase flow (http://calypso.localhost:3000/domains/add/[domain]/google-apps/[domain])
3. Check that the designs look as above

#### Reviews
  
- [ ] Code
- [ ] Product

### CODE REVIEW

File: client/components/upgrades/google-apps/google-apps-dialog/index.jsx [hunk 1]
```diff
@@ -97,7 +110,16 @@ const GoogleAppsDialog = React.createClass( {
 		return (
 			<header className="google-apps-dialog__header">
 				<h2 className="google-apps-dialog__title">
-					{ this.translate( 'Add Professional Email to %(domain)s', { args: { domain: this.props.domain } } ) }
+					{
+						this.translate(
+							'Add Professional email from Google to %(domain)s',
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/index.jsx]: :disappointed: 25 existing translations will be lost with this change.

File: client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+						{ translate( '%(annualPrice)s Billed yearly — get 2 months free!', {
+							args: { annualPrice: this.props.annualPrice }
+						} ) }
+					</h5>
+				</div>
+
+				<div className="google-apps-dialog__product-features">
+					<h5 className="google-apps-dialog__product-feature">
+						<img src="/calypso/images/g-suite/logo_gmail_48dp.svg" />
+						<p>{ translate( 'Professional email address' ) }</p>
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Professional Email Address' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+						{
+							translate(
+								"We've partnered with Google to offer you email, storage, docs, calendars " +
+								'and more, integrated with your site.'
+							)
+						}
+					</p>
+
+					<h4 className="google-apps-dialog__price-per-user">
+						{ translate( '%(monthlyPrice)s per user / month', { args: { monthlyPrice: this.props.monthlyPrice } } ) }
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( '%(price)s per user / month ' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx [hunk 4]
```diff
@@ -28,134 +28,174 @@ const AddGoogleAppsCard = React.createClass( {
...
-					</div>
-
-					{ this.renderAddGoogleAppsButton() }
-				</header>
+							<div className="add-google-apps-card__price">
+								<h4 className="add-google-apps-card__price-per-user">
+									<span>
+										{
+											this.translate(
+												'{{strong}}%(price)s{{/strong}} per month / user',
```
Comment (a8ci18n) [inline client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( '{{strong}}%(price)s{{/strong}} per user / month' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx [hunk 5]
```diff
@@ -28,134 +28,174 @@ const AddGoogleAppsCard = React.createClass( {
...
+							{ /* Intentionally not translated */ }
+							<strong>G Suite</strong>
+						</h3>
+					</header>
+				</CompactCard>
+				<CompactCard>
+					<div className="add-google-apps-card__product-details">
+						<div className="add-google-apps-card__description">
+							<h2 className="add-google-apps-card__title">
+								{ this.translate( 'Professional email and so much more.' ) }
```
Comment (a8ci18n) [inline client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx]: Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Professional Email & More' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/upgrades/google-apps/google-apps-dialog/index.jsx [hunk 6]
```diff
@@ -97,10 +110,19 @@ const GoogleAppsDialog = React.createClass( {
...
+								args: {
+									domain: this.props.domain
+								}
+							}
+						)
+					}
 				</h2>
 				<h5 className="google-apps-dialog__no-setup-required">
-					{ this.translate( 'No setup or software required, easy to manage from your dashboard' ) }
+					{ this.translate( 'No setup or software required. Easy to manage from your dashboard.' ) }
```
Comment (a8ci18n) [inline client/components/upgrades/google-apps/google-apps-dialog/index.jsx]: :ok: This change is safe, we'll keep the existing 24 translations.

File: client/components/upgrades/google-apps/google-apps-dialog/index.jsx [hunk 7]
```diff
@@ -47,27 +48,39 @@ const GoogleAppsDialog = React.createClass( {
 
 	render() {
 		const gapps = this.props.productsList && this.props.productsList.get().gapps;
-		let price = gapps && gapps.cost_display;
+		const price = gapps && gapps.cost_display;
 
 		// Gapps price is stored annually but we'd like to show a monthly price
-		price = price && price.replace( /(\d+\.?\d+)/, ( value ) => {
+		const monthlyPrice = price && price.replace( /(\d+\.?\d+)/, ( value ) => {
```
Comment (klimeryk) [inline client/components/upgrades/google-apps/google-apps-dialog/index.jsx]: Just a quick comment before I dig into this:
This (and line 59) might be suspectible to the same issue that I thought I fixed in https://github.com/Automattic/wp-calypso/pull/13527, but obvisously, missed this place 😞 Something like `\d+[.,]?\d+` should fix it 👍

Comment (scruffian) [inline client/components/upgrades/google-apps/google-apps-dialog/index.jsx]: Might be worth abstracting it to a function

**Comment by danhauk** [comment]
@klimeryk There is a noticeable delay but I think it's just from fetching the data. [I'm seeing the same delay on staging/production](https://cloudup.com/ctlFdJTqtkW). I just pushed a commit to add a placeholder to make the delay and transition less jarring, like we have now.

**Comment by klimeryk** [comment]
Thanks, @danhauk, it looks much better now 👍 
I've fixed the inconsistent copy for price per user / month and moved the calculations for monthly/annual prices to one place.

LGTM 👍

**Comment by aidvu** [comment]
Feel free to merge. Made a maintenance task to refactor and get rid of linter errors.

**Pull request merged**